Dockerfile

```diff
@@ -19,7 +19,7 @@ COPY cmake /usr/src/forgottenserver/cmake/
 COPY src /usr/src/forgottenserver/src/
 COPY CMakeLists.txt /usr/src/forgottenserver/
 WORKDIR /usr/src/forgottenserver/build
-RUN cmake .. && make
+RUN cmake .. && make -j $(($(nproc) / 2 + 1))
```

Comment:
is that intended or pushed by accident?

Reply:
Intended, pushed as separate commit: https://github.com/otland/forgottenserver/pull/3584/commits

Reply:
I'd rather pass `MAKEFLAGS` as an env/arg and not try to anything smart _inside_ the container.